npm-shrinkwrap.json

```diff
@@ -13,12 +13,12 @@
       "resolved": "https://registry.npmjs.org/bignumber.js/-/bignumber.js-1.5.0.tgz"
     },
     "bluebird": {
-      "version": "2.9.12",
-      "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-2.9.12.tgz"
+      "version": "2.9.24",
```

Comment:
It would be helpful if the commit message included which dependencies you updated, since the shrinkwrap will have all the child dependency updates and it's difficult to parse out.